grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -1018,6 +1018,11 @@ class UrlMappings {
             action = [GET: "details"]
         }
 
+        "/api/mobile/products/$id/barcode" {
```

Comment:
There was a change request related to this endpoint in mobile repo

Reply:
https://github.com/openboxes/openboxes-mobile/pull/346/commits/c68a3f87ec52c4f6055206d3792d8b01a0a5c948